cms/templates/js/highlights-editor.underscore

```diff
@@ -19,7 +19,10 @@
         'read our {linkStart}documentation{linkEnd}.'
     ),
     {
-        linkStart: edx.HtmlUtils.HTML('<a href="' + highlights_doc_url + '">'),
+        linkStart: edx.HtmlUtils.interpolateHtml(
+            edx.HtmlUtils.HTML('<a href={highlightsDocUrl}">'),
+            {highlightsDocUrl: highlights_doc_url},
+        ),
```

Comment:
Although this change is fine, I don't believe it's needed - as there is no translation nor HTML escaping needed here.

Reply:
I know we control the doc URL, but isn't it good practice to HTML escape it?